Add decompress feature

t3rry327 opened the pull request:
see Readme.MD

rexlManu commented:
Hi, thanks for your constribution.

I would suggest to replace the decompress input as a boolean like `decompress-target` and it should check if the target is a archive file.

t3rry327 commented:
@rexlManu Thanks for the suggestion.

In this Pull Request, I've introduced the capability for automatic decompression of uploaded archive files `(.zip, .tar, .tar.gz, .tgz, .rar)` directly on the server-side. This feature can be enabled by setting a new input parameter "`decompress-target`" to true in the configuration.

I've added a function '`decompressFile`' which interfaces with the Pterodactyl API's decompress endpoint, triggering the decompression process on the server upon upload of an archive file.

To ensure that only valid archive files are decompressed, I've implemented an '`isArchiveFile`' check that evaluates the file extension of the uploaded file, and returns true if the file is of a recognized archive format.
